templates/build.gradle

```
@@ -190,24 +187,26 @@ task createSkeleton() {
 
 task cacheReflections {
     description = 'Caches reflection output to make regular startup faster. May go stale and need cleanup at times.'
-    // TODO: The extra "org" qualifier excludes test classes otherwise sucked up in Jenkins, causing issues. Redo later
-    File dirToReflect = new File(sourceSets.main.output.classesDir, "org")
-    inputs.files dirToReflect
-    outputs.file file(sourceSets.main.output.classesDir.toString() + "/reflections.cache")
+    inputs.files sourceSets.main.output.classesDirs,
+            // getClassesDir from all sourceSets (for any jvm (seems) language)
+            configurations."${sourceSets.main.runtimeClasspathConfigurationName}"
+
+    outputs.upToDateWhen {classes.state.upToDate}
+    outputs.file("$buildDir/resources/main/reflections.cache")
```

Where is `$buildDir/resources/main/reflections.cache` created? It seems that the reflections are saved in a different location. Did you mean to use `$buildDir/classes/reflections.cache` instead?
```suggestion
    outputs.file("$buildDir/classes/reflections.cache")
```